### PR TITLE
Complete the readme with docker-less installation and launching

### PR DESCRIPTION
**Pull request by j33f**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,33 @@ Where `my-kuzzle-container-name` is the container name where your Kuzzle is runn
 
 The container name is in the last column.
 
+## Without docker with an already running Kuzzle
+
+### Prerequisites
+
+* A properly installed [nodeJs](https://nodejs.org/en/download/package-manager/) **version 4** or upper
+* Las version of [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
```
Comment (AnthonySendra) [inline README.md]: Las -> Last

File: README.md [hunk 2]
```diff
@@ -47,6 +47,33 @@ Where `my-kuzzle-container-name` is the container name where your Kuzzle is runn
...
+* Last version of [RubyGems](https://rubygems.org/pages/download)
+
+### First step
+
+You'll need to clone the back-office repository then install the dependencies and compile the css files:
+
+	$ git clone https://github.com/kuzzleio/kuzzle-bo.git && cd kuzzle-bo
+	$ npm install 
+	$ bower install --config.interactive=false
+	$ grunt sass
```
Comment (AnthonySendra) [inline README.md]: Missing lines in "Prerequisites" about installing Sass ? And Bower ?

Comment (j33f) [inline README.md]: right ! should'nt they be in the package.json dependencies ?

Comment (AnthonySendra) [inline README.md]: Nop, should be installed globally ;)

**Comment by AnthonySendra** [comment]
+1

**Pull request merged**